libraries/chain/transaction_context.cpp

```diff
@@ -70,7 +70,7 @@ namespace eosio { namespace chain {
          }
       }
 
-      if( billed_cpu_time_us > 0 )
+      if( explicit_billed_cpu_time || billed_cpu_time_us > 0 )
```

Comment:
This is not needed as `validate_cpu_usage_to_bill` is just going to validate `billed_cpu_time_us` is <= than `objective_duration_limit`